pkg/cloudprovider/aws/factory.go

```diff
@@ -92,6 +92,12 @@ func (f *Factory) CapacityFor(provisioner *v1alpha1.Provisioner) cloudprovider.C
 	}
 }
 
+func (f *Factory) TerminateFor() cloudprovider.Termination {
+	return &Termination{
```

Comment:
I think this should potentially be called `Terminator` instead of `Termination`? To match `Provisioner`, etc.

Reply:
I think Termination fits here as the counterpart is Capacity, rather than Provisioner.